classes/patterns/class-action.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
```

Comment:
Just thinking.. shouldn't we rename the `class` for a another more specific name? SInce `Action` sounds a general stuff. Maybe `ActionLayout`?

Reply:
Maybe all of these pattern layouts' classes should indicate it in their name, so here for example `ActionPatternLayout`? 🤔

Reply:
I agree to change to `ActionPatternLayout`. Not now, but we should rename the other layout patterns too.